app/src/main/java/lt/vilnius/tvarkau/fragments/BaseFragment.kt

```diff
@@ -58,19 +71,42 @@ abstract class BaseFragment : Fragment(), OnBackPressed {
     override fun onResume() {
         super.onResume()
 
-        analytics.trackOpenFragment(activity, javaClass.simpleName)
+        screen?.run {
+            if (titleRes != 0) {
+                activity.setTitle(titleRes)
+            }
+
+            trackingScreenName.emptyToNull()?.let {
```

Comment:
You have a choice if you want to track fragment or not. If name is empty string tracking is disabled.